tensorflow_datasets/image/imagewang_test.py

```diff
@@ -0,0 +1,19 @@
+""" Tests for Imagewang contains Imagenette and Imagewoof combined. """
```

Comment:
Consider creating 3 different tests (different classes) for the 3 variants of the dataset. Usually 1 file for each split is enough.

Reply:
Use `BUILDER_CONFIG_NAMES_TO_TEST` and set it equal to `['<name of builder config>']` when implementing the classes.

Reply:
Is this better: https://github.com/tensorflow/datasets/pull/1326/commits/3f0754ecb8fca4a575f93b82e514e0dd945be22a?

The tests fail because there is no test split. The reason for that is because Imagewang does not have a publicly available test set.